v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/KafkaWriteOptions.java

```diff
@@ -44,7 +44,10 @@ public interface KafkaWriteOptions extends PipelineOptions {
         @TemplateParameter.TemplateEnumOption(KafkaAuthenticationMethod.TLS),
         @TemplateParameter.TemplateEnumOption(KafkaAuthenticationMethod.NONE)
       },
-      helpText = "Type of authentication mechanism to use with the destination Kafka.")
+      helpText =
+          "Mode of authentication with the Kafka cluster. "
+              + "NONE for no authentication, SASL_PLAIN for SASL/PLAIN username/password, "
+              + "TLS for certificate-based authentication.")
```

Comment:
```suggestion
              + " `TLS` for certificate-based authentication.")
```

Reply:
Do we need to include that Apache Kafka for BigQuery only supports the SASL_PLAIN authentication mode?

Reply:
Yes.